UPGRADE.md

```diff
@@ -86,6 +86,9 @@ After:
 
 ### Order / OrderBundle
 
+* Method `findOrdersUnpaidSince` from `Sylius\Bundle\OrderBundle\Doctrine\ORM\OrderRepository` has been moved to `Sylius\Bundle\CoreBundle\Doctrine\ORM\OrderRepository` as it has been depended on Core component. 
+ If you have used `findOrdersUnpaidSince` method you had to require CoreBundle anyway. If you haven't used this method, but you have been force to require `CoreBundle` than you can remove this dependency. 
```

Comment:
`If you haven't been using this method, you can remove the 'CoreBundle' dependency.`
instead of those two, wdyt?